pkg/kn/commands/plugin/plugin_handler.go

```diff
@@ -0,0 +1,109 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+)
+
+// PluginHandler is capable of parsing command line arguments
+// and performing executable filename lookups to search
+// for valid plugin files, and execute found plugins.
+type PluginHandler interface {
+	// exists at the given filename, or a boolean false.
+	// Lookup will iterate over a list of given prefixes
```

Comment:
"given prefixes"? Where is this list? none of the args to Lookup are slices

Reply:
Hardcoded. There is a var for it

Reply:
I should have been more clear. I think the comment needs to change because the phrase "given prefixes" (to me) implies "passed in prefixes" , meaning as an arg.  if it said "..iterate of the list in ValidPrefixes" then it would have been more clear.